packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -945,7 +952,8 @@ class _PageViewState extends State<PageView> {
         controller: widget.controller,
         physics: physics,
         restorationId: widget.restorationId,
-        scrollBehavior: widget.scrollBehavior ?? ScrollConfiguration.of(context).copyWith(scrollbars: false),
+        // Adds compatability for mouse-based scroll/drag on the PageView for mouse-based platforms by implementing the customDragDevices to the dragDevices parameter.
+        scrollBehavior: widget.scrollBehavior ?? ScrollConfiguration.of(context).copyWith(scrollbars: false, dragDevices: customDragDevices),
```

Comment:
I don't think we need to hardcode this into the widget. You can already provide a custom ScrollConfiguration via `scrollBehavior` if you need this behavior.

Reply:
As Flutter expands into the platforms of Desktop and Web, I really hope that it is the basic application of a PageView widget to allow the user to scroll within the widgets provided as children which is broken on the Web and the Desktop due to the above factor. I hope that this inclusion of this hardcode into the widget can help many people seamlessly achieve the same without much of a struggle and can save them a lot of time if it was included out of the box! When I was recently developing my application, Aham Svastha for the cross-platform for the first time as I recently shifted to Flutter from native iOS and Android development, I felt it could be great if I hadn't need to search for "how to scroll pageview flutter desktop" on the web before arriving to the solution of adding scroll behaviour and custom implementing it. I felt by incorporating this change could help many of those fresh Flutter developers have a good start! Hoping to hear a positive reply from you soon!
Thanking you
Manas Malla
Flutter Developer
Apple's WWDC20 Swift Student Challenge Winner
Google DevFest India Quiz Winner